x-pack/plugins/siem/public/components/timeline/body/renderers/auditd/generic_file_details.tsx

```diff
@@ -129,8 +129,8 @@ export const AuditdGenericFileDetails = pure<GenericDetailsProps>(
   ({ data, contextId, text, fileIcon = 'document' }) => {
     const id = data._id;
     const session: string | null | undefined = get('auditd.session[0]', data);
-    const hostName: string | null | undefined = get('host.name', data); // TODO: Array host.name[0]
-    const userName: string | null | undefined = get('user.name', data); // TODO: Array user.name[0]
+    const hostName: string | null | undefined = get('host.name[0]', data); // TODO: Array host.name[0]
```

Comment:
Leftover `TODO`

Reply:
Yep, and I found a few spots I missed pull things out of an array that I also fixed and am pushing up